### DISCUSSION TITLE
Show snake info panel on mobile

### DISCUSSION THREAD

**Discussion by jlafayette** [BattlesnakeOfficial/feedback#145] [Feature Requests]
I was watching a game on my phone and noticed that there doesn't seem to be a way to see the snake names when viewing. Could we add that below the game board on mobile?

Screenshot from the phone:
![Screenshot_20211206-082851](https://user-images.githubusercontent.com/17697135/144885417-df461167-36f5-42ef-bb38-38144c2a7b43.png)
Screenshot of the snake info panel that is shown on the desktop version:
![Screenshot 2021-12-06 8 33 30 AM](https://user-images.githubusercontent.com/17697135/144885514-433498c4-4fde-4d12-982e-707a42a8c222.png)

**Comment by originalwebgurl** [comment]
Thanks for the feedback @jlafayette! 

We do have something in the works to improve the playback experience on mobile devices. It's not a quick fix however so will take some time to get right. I'll continue to update you in this thread on it's progression.
